### PR TITLE
[Issue #182] PEM action added

### PR DESCRIPTION
**Pull request by aafa**
Simple implementation via console call

Related issue #182

### CODE REVIEW

File: lib/fastlane/actions/pem.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module Fastlane
+  module Actions
+
+    class PemAction
+      def self.run(params)
+        unless Helper.test?
+            raise 'pem is not installed, please install using `[sudo] gem install pem`'.red if `which pem`.length == 0
```
Comment (KrauseFx) [inline lib/fastlane/actions/pem.rb]: `PEM` is actually a dependency of `fastlane`, we don't need this check :+1:

**Comment by KrauseFx** [comment]
I'll try to work out a solution, that doesn't use the console call.

**Comment by KrauseFx** [comment]
And here we go: https://github.com/KrauseFx/fastlane/commit/80a9d0a43068eee3e75e579379fcfc18e7c49298

Thanks again for your work!

**Comment by aafa** [comment]
Wow, looks great!  Thank YOU for your amazing work! :+1:  :smiley:

**Comment by KrauseFx** [comment]
Thanks for pushing me to work on this. I think the `PEM` integration is great now :+1: 

I haven't yet released the update of `fastlane`, but you know how to install it locally in the mean time :wink:

**Pull request closed without merging**